conf/application.conf

```diff
@@ -137,6 +137,7 @@ urls {
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   continue = "continue"
   signOut = "https://www.gov.uk/charity-recognition-hmrc"
+  signedYouOut = "https://www.gov.uk/charity-recognition-hmrc/error/signed-you-out"
```

Comment:
have to check with Blaine whether the url is correct or not

Reply:
That is the URL specified in the ticket